internal/protocoltest/awsrestjson/api_op_JsonMaps.go

```diff
@@ -27,11 +27,11 @@ func (c *Client) JsonMaps(ctx context.Context, params *JsonMapsInput, optFns ...
 }
 
 type JsonMapsInput struct {
-	MyMap map[string]types.GreetingStruct
+	MyMap map[string]*types.GreetingStruct
 }
 
 type JsonMapsOutput struct {
-	MyMap map[string]types.GreetingStruct
+	MyMap map[string]*types.GreetingStruct
 
```

Comment:
Looks like this change also includes changes to the smithy protocol test as well. Were these maps/list updated to be `@sparse`?

Reply:
Yup, looks like master tests were updated to have sparse trait.